nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -127,7 +127,7 @@ <h1>
             <div class="row">
                 <div class="col-md-6">
                     {% block content_left_page %}{% endblock content_left_page %}
-                    {% include 'inc/custom_fields/panel.html' with custom_fields=object.get_custom_field_groupings_basic computed_fields_advanced_ui=False %}
+                    {% include 'inc/custom_fields/panel.html' with custom_fields=object.get_custom_field_groupings_basic computed_fields=object.get_computed_fields_grouping_basic computed_fields_advanced_ui=False %}
```

Comment:
Isn't a similar change needed in `nautobot/core/templates/inc/object_details_advanced_panel.html` for the "advanced_ui" computed-fields?

Also, because they override the entire `content` block, you'll need to also make this specific update to `nautobot/dcim/templates/dcim/device.html` and `nautobot/dcim/templates/dcim/softwareimagefile_retrieve.html`.

Reply:
Updated